### ISSUE TITLE
A Gentle Introduction to Cross-Entropy for Machine Learning

### ISSUE THREAD

**Issue by ikedaosushi** [ikedaosushi/tech-news#18433]
A Gentle Introduction to Cross-Entropy for Machine Learning<br>
Cross-entropy is commonly used in machine learning as a loss function. Cross-entropy is a measure from the field of information theory, building upon entropy and generally calculating the difference between two probability distributions.<br>
https://ift.tt/2J9aQ49
